src/middleware/github-server-app-middleware.ts

```diff
@@ -2,12 +2,20 @@ import { NextFunction, Request, Response } from "express";
 import { GitHubServerApp } from "models/github-server-app";
 import { Installation } from "models/installation";
 
+export type GitHubAppReqLocals<T = Record<string, any>> = T & {
+	gitHubAppConfig: {
+		gitHubAppId: number | undefined;
```

Comment:
First thing first, which one are we using?

Reply:
using for what?